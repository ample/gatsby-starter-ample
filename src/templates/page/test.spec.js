import React from "react"
import renderer from "react-test-renderer"

import PageTemplate from "./"

describe("PageTemplate", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<PageTemplate sections={[]} />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
