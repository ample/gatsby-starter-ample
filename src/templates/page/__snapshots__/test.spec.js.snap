// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PageTemplate renders correctly 1`] = `
Array [
  <header
    className="header"
  >
    <div>
      <div
        className="logo_container"
      >
        <a
          className="logo"
          href="/"
        >
          <svg
            viewBox="0 0 421.309 280.566"
          >
            <g>
              <path
                d="M210.264 122.508c-11.666 2.693-18.88 15.775-17.773 30.7a41.244 41.244 0 00.963 6.417c4.026 17.424 18.072 29.075 31.385 25.999 13.322-3.076 20.836-19.702 16.814-37.124a41.598 41.598 0 00-1.933-6.196c-5.551-13.899-17.778-22.492-29.456-19.796zM59.47 204.185l-2.19-9.566s-16.54 8.767-22.43 11.82c-5.894 3.06-9.14 8.505-8 13.44 2.02 8.754 10.188 8.424 16.959 6.86 10.941-2.526 17.952-12.618 15.662-22.554zm199.366-22.944c-.043.089-.085.186-.136.28-6.087 12.009-16.586 21.116-29.909 24.194a43.516 43.516 0 01-25.698-2.026l14.022 60.723a13.242 13.242 0 01-25.805 5.958l-15.224-65.943-10.343-44.802-2.48-10.738-.034.008a15.098 15.098 0 00-29.405 6.789l-.037.009 12.805 55.467c.006.023.007.044.013.07l-25.795 5.958c-.006-.026-.016-.044-.022-.07l-12.807-55.47-.052.012a15.089 15.089 0 00-29.387 6.784l-.045.014 12.802 55.468c.006.025.007.043.013.07L15.807 241.43A29.572 29.572 0 014.9 224.687c-2.39-10.34 1.126-20.796 8.49-28.434a37.315 37.315 0 0110.862-7.68l28.574-13.622-1.08-4.66c-.736-3.192-6.11-9.113-16.324-6.755-15.068 3.481-9.45 18.81-20.777 21.428a11.97 11.97 0 01-14.371-8.932 11.633 11.633 0 01.1-5.43c3.328-12.736 14.835-23.746 30.366-27.332 11.747-2.712 23.317-.595 31.964 4.916a34.123 34.123 0 0153.646-9.132 34.113 34.113 0 0157.904-9.972c5.91-12.944 16.816-22.824 30.824-26.061 24.234-5.595 48.852 10.665 56.826 36.693.403 1.323.795 2.652 1.107 4.017.428 1.854.75 3.71 1.005 5.55.062.535.08 1.064.137 1.585.143 1.331.268 2.652.312 3.966.018.566.014 1.116 0 1.687a55.242 55.242 0 01-.077 3.813c-.026.51-.089 1.015-.138 1.53a54.032 54.032 0 01-.464 3.901c-.07.422-.15.847-.234 1.273a57.86 57.86 0 01-.894 4.048c-.094.33-.188.635-.267.968a52.333 52.333 0 01-1.387 4.208c-.073.214-.162.422-.231.623a51.362 51.362 0 01-1.938 4.358zm36.988-40.153c.028.11.028.219.05.338l7.807 33.695a.339.339 0 00.013.055l-26.388 6.095a.203.203 0 00-.02-.06l-7.856-34.03-14.598-63.245-15.993-69.26A13.814 13.814 0 01249.44.35a13.483 13.483 0 0115.565 8.006zm39.456-46.177c-8.366 3.722-13.957 12.827-14.76 23.56l45.903-10.594c-5.414-9.301-14.437-15.032-23.487-14.716a20.914 20.914 0 00-3.968.506 20.3 20.3 0 00-3.688 1.243zm53.348 48.658a10.238 10.238 0 01-.168 5.297 52.14 52.14 0 01-4.295 8.238l-60.323 13.93a56.093 56.093 0 01-24.683-35.263c-6.588-28.55 8.634-56.523 33.95-62.368 25.315-5.844 51.264 12.62 57.856 41.173.117.508.215 1.026.298 1.54l.104.596a7.923 7.923 0 01-5.906 8.575l-4.508 1.056-8.05 1.858-49.243 11.369c6.072 11.888 17.487 18.348 28.382 15.836 6.893-1.591 12.5-6.515 15.77-13.855a11.258 11.258 0 017.575-6.026 10.962 10.962 0 0113.241 8.044zm10.588 11.835a11.189 11.189 0 1113.412 8.387 11.173 11.173 0 01-13.412-8.387z"
                fill="#fff"
              />
            </g>
          </svg>
        </a>
        <button
          className="menu_button"
          onClick={[Function]}
        >
          <span>
            Menu
          </span>
          <svg
            height={19}
            viewBox="0 0 448 512"
            width={24}
          >
            <path
              d="M442 114H6a6 6 0 01-6-6V84a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6zm0 160H6a6 6 0 01-6-6v-24a6 6 0 016-6h436a6 6 0 016 6v24a6 6 0 01-6 6z"
            />
          </svg>
        </button>
      </div>
      <div
        className="navigation"
      >
        <ul
          className="link_list top_navigation"
        >
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Duis
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Cupidatat
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Ullamco
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Labore
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Consectetur
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Pariatur
            </a>
          </li>
        </ul>
        <ul
          className="link_list main_navigation"
        >
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Reprehenderit
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Officia sint
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Veniam 
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Lacinia dolor
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Consequat 
            </a>
          </li>
          <li>
            <a
              href="#"
              rel="noopener"
              target="_blank"
            >
              Voluptate nostrud
            </a>
          </li>
        </ul>
      </div>
    </div>
  </header>,
  <main>
    <div
      className="page"
    />
  </main>,
  <footer
    className="footer"
  >
    <div>
      <div
        className="logo"
      >
        <a
          href="/"
        >
          <svg
            viewBox="0 0 292.642 340.529"
          >
            <g>
              <g>
                <path
                  d="M183.017 315.94c-3.61-15.747 6.689-33.17 25.385-42.987 18.688-9.808 71.15-37.94 71.15-37.94l6.924 30.535c7.247 31.714-15.002 64.002-49.693 72.163-21.475 5.048-47.37 6.178-53.766-21.771zM111.27 323.753a91.566 91.566 0 005.675 16.776 157.222 157.222 0 014.916-305.155L275.019 0l17.623 76.35c-27.351-16.405-63.045-22.577-99.318-14.165-49.635 11.526-86.344 46.89-96.898 87.81a37.41 37.41 0 00-.275 17.44 38.338 38.338 0 0046 28.69c36.188-8.401 18.127-57.647 66.273-68.83 32.638-7.577 49.863 11.45 52.235 21.703l3.484 14.981-91.261 43.758a118.729 118.729 0 00-34.66 24.665c-23.498 24.543-34.659 58.132-26.953 91.351z"
                  fill="#fff"
                />
              </g>
            </g>
          </svg>
        </a>
      </div>
      <ul
        className="link_list is_vertical"
      >
        <li>
          <strong>
            Quisque
          </strong>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Vestibulum vitae
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Pellentesque tempor venenatis
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Fusce molestie
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Etiam in lacinia dolor
          </a>
        </li>
      </ul>
      <ul
        className="link_list is_vertical"
      >
        <li>
          <strong>
            Vestibulum
          </strong>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Fusce molestie
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Pellentesque venenatis
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Vestibulum vitae
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            lacinia dolor
          </a>
        </li>
      </ul>
      <nav
        className="social_nav"
      >
        <a
          href="#"
          rel="noopener"
          target="_blank"
        >
          <svg
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M459.37 151.716c.325 4.548.325 9.097.325 13.645 0 138.72-105.583 298.558-298.558 298.558-59.452 0-114.68-17.219-161.137-47.106 8.447.974 16.568 1.299 25.34 1.299 49.055 0 94.213-16.568 130.274-44.832-46.132-.975-84.792-31.188-98.112-72.772 6.498.974 12.995 1.624 19.818 1.624 9.421 0 18.843-1.3 27.614-3.573-48.081-9.747-84.143-51.98-84.143-102.985v-1.299c13.969 7.797 30.214 12.67 47.431 13.319-28.264-18.843-46.781-51.005-46.781-87.391 0-19.492 5.197-37.36 14.294-52.954 51.655 63.675 129.3 105.258 216.365 109.807-1.624-7.797-2.599-15.918-2.599-24.04 0-57.828 46.782-104.934 104.934-104.934 30.213 0 57.502 12.67 76.67 33.137 23.715-4.548 46.456-13.32 66.599-25.34-7.798 24.366-24.366 44.833-46.132 57.827 21.117-2.273 41.584-8.122 60.426-16.243-14.292 20.791-32.161 39.308-52.628 54.253z"
            />
          </svg>
        </a>
        <a
          href="#"
          rel="noopener"
          target="_blank"
        >
          <svg
            viewBox="0 0 320 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M279.14 288l14.22-92.66h-88.91v-60.13c0-25.35 12.42-50.06 52.24-50.06h40.42V6.26S260.43 0 225.36 0c-73.22 0-121.08 44.38-121.08 124.72v70.62H22.89V288h81.39v224h100.17V288z"
            />
          </svg>
        </a>
      </nav>
      <ul
        className="link_list"
      >
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Privacy Policy
          </a>
        </li>
        <li>
          <a
            href="#"
            rel="noopener"
            target="_blank"
          >
            Terms of Service
          </a>
        </li>
      </ul>
      <small
        className="copyright"
      >
        © 
        2020, All Rights Reserved
      </small>
    </div>
  </footer>,
]
`;
